docs/source/development/documentation/api_generation.md

```diff
@@ -0,0 +1,56 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  main_language: python
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.8
+---
+
+# Rendered HTML for docstring example
```

Comment:
Find the difference between this section title and it's name in the side bar "API Generation" a little bit jarring

Reply:
Also as far as I can tell this page doesn't actually show the rendered output, which the link from `docstring_style.md` suggested it would

Reply:
OK -  RTD runs `sphinx-build` from a different location. Just needed an alternative path in `conf.py` when running on RTD (or we could update where we build from...). All rendered now.

Reply:
That renders now, but it seems to have broken the nesting of the API reference in the side bar

Reply:
You mean the folding listings of content with files? So it has. 🤯 

I _much_ prefer the MkDocs explicit table of contents tree than `sphinx` ToC directives, which are just baffling.